### DISCUSSION TITLE
Ability to view my starred themes

### DISCUSSION THREAD

**Discussion by corysimmons** [jln13x/ui.jln.dev#3] [Ideas]
I would love to "Explore" then star a bunch of pretty themes.

But there's no way for me to access them in one spot after that. Can there be a `/my-stars` route that lists cards of all the palettes I've starred please?

**Comment by jln13x** [comment]
#5 Done, let me know what you think :)

**Comment by corysimmons** [comment]
Uh oh, I think that broke your build.

![image](https://github.com/jln13x/ui.jln.dev/assets/224636/71e42dc6-3cab-4a9a-856c-d9b89bb7ea3b)

**Reply by jln13x** [reply]
👀  Working on it, rolled back for now

**Comment by corysimmons** [comment]
@jln13x Both your changes are live and work wonderfully! Thank you! Using this from now on!
